buf/registry/module/v1beta1/label_service.proto

```diff
@@ -144,6 +144,12 @@ message ListLabelHistoryRequest {
   uint32 page_size = 1 [(buf.validate.field).uint32.lte = 250];
   // The page to start from.
   //
+  // The value may be a page_token returned from ListLabelHistoryResponse,
+  // or it may be a Commit id.
+  //
+  // If a Commit id, the returned history will start at that Commit.
```

Comment:
Special-casing page_token for a single endpoint makes me extremely nervous on a bunch of levels, can discuss

Reply:
Changed to start_commit_id